packages/shared/src/components/modals/PostToSquadModal.tsx

```diff
@@ -56,9 +56,12 @@ function PostToSquadModal({
   const onPostSuccess = async (squadPost?: Post) => {
     if (squadPost) onSharedSuccessfully?.(squadPost);
 
-    displayToast(
-      'This post is being processed and will be shared with your Squad shortly',
-    );
+    if (isLink) {
+      displayToast(
+        'This post is being processed and will be shared with your Squad shortly',
+      );
+    }
```

Comment:
Ah missed that variable when checking 😂
Let's do it something like this instead:

```suggestion
      displayToast(isLink ? 'This post is being processed and will be shared with your Squad shortly' : 'This post has been shared to your squad');
```